src/py21cmfast/outputs.py

```diff
@@ -936,10 +936,23 @@ def save(self, fname=None, direc=".", clobber: bool = False):
         return self._write(direc=direc, fname=fname, clobber=clobber)
 
     @classmethod
```

Comment:
I had a question here. Where does the Coeval object read in the parameter structures? The Coeval `__init__` takes them directly from the `BrightnessTemp` struct, but these don't seem to be saved with the structures looking at the `_write_particulars` function.